kolibri/core/assets/src/api-resources/contentNode.js

```diff
@@ -87,6 +87,18 @@ class ContentNodeResource extends Resource {
     }
     return promise;
   }
+  getAllContentCollection(resourceIds = {}, getParams = {}) {
```

Comment:
Special method to return a collection that queries the `all_content` list endpoint.